File: src/easy/CommonFactors.swift
func commonFactors(_ a: Int, _ b: Int) -> Int {
  var count = 0
  let m = max(a, b)
  for i in 1..<m {
    if a % i == 0 && b % i == 0 {
      count += 1
    }
  }

  return count
}

File: lib/easy/common_factors.dart
import "dart:math";

int commonFactors(int a, int b) {
  int count = 0;
  int m = max(a, b);
  for (int i = 1; i < m + 1; i++) {
    if (a % i == 0 && b % i == 0) {
      count++;
    }
  }

  return count;
}
